src/pages/DokumentOversikt/components/statuspanel/statuspanel.less

```diff
@@ -0,0 +1,159 @@
+@import '../../../../basic.less';
+
+@space-to-line-horisontal: calc(50% + 1.5rem);
+@space-to-line-vertical: calc(50% + 1.5rem);
+@border: 1px solid #000;
+@top-horisontal: calc(17%);
+@left-vertical: 1rem;
+@width-horizontal: 50%;
+@height-vertical: 40%;
+
+.circle {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 2rem;
+    height: 2rem;
+    border-radius: 50%;
+    border: 1px solid black;
```

Comment:
Skissene bruker fargen #CCC, med bredde 2px.

Reply:
den fargen ble veldig dus, og bredden virker litt tykk. beholder det slik det er inntil videre

Reply:
ser ut som fargen i abstract er #78706a for sirkel og #979797 for strek